osidb/models.py

```diff
@@ -1743,6 +1745,18 @@ def _validate_public_source_no_ack(self):
 class FlawCommentManager(ACLMixinManager, TrackingMixinManager):
     """flawcomment manager"""
 
+    def pending(self):
+        """
+        Get new pending comments that are about to be submitted to bugzilla.
+
+        Note that consumers of this might break if ATOMIC_REQUESTS is not used
+        because long-forgotten uncleaned pending comments might suddenly be
+        posted to bugzilla.
+        """
+        pending_comments = super().get_queryset().filter(external_system_id="")
+
+        return pending_comments
```

Comment:
nitpick: Why not just `return super().get_queryset().filter(external_system_id="")` ?

Reply:
Very slightly easier debugging but will do as you suggest.

Reply:
Maybe just a personal preference :smile:

Reply:
Resolved in https://github.com/RedHatProductSecurity/osidb/pull/209/commits/f9ebfa5badd141532da083c3d6506b796e4868b4